frontend/public/components/cluster-settings/oauth.tsx

```diff
@@ -74,7 +74,7 @@ const OAuthDetails: React.SFC<OAuthDetailsProps> = ({obj}: {obj: OAuthKind}) =>
     <div className="co-m-pane__body">
       <SectionHeading text="Identity Providers" />
       <p className="co-m-pane__explanation co-m-pane__explanation--alt">
-        Identity providers determine how users log into the cluster.
+        Identity providers determine how users log into the cluster. To specify how new identities are mapped to logged in users the mapping method is always set to the value: claim.
```

Comment:
We can't make this statement since other mapping methods can be provided via YAML or outside console.

Reply:
@spadgett maybe thats what we say here then? The forms below assume mapping method set to claim. Other methods can be used via yaml ....

Reply:
I'm not sure we really need to say anything. You should almost always be using claim. We can make the UI opinionated IMO since there's always a YAML fallback for advanced users.

Reply:
@spadgett yeah but the bug does't say the ui can't be opinionated, rather, it should state how so. as of now, we only tell the user after the fact.

Reply:
If we're going to add it, it should be on each of the forms. I'm not sure I agree we really need to say it.

@enj opinion?